parsl/providers/torque/torque.py

```diff
@@ -76,7 +76,8 @@ def __init__(self,
                  max_blocks=100,
                  parallelism=1,
                  launcher=AprunLauncher(),
-                 walltime="00:20:00"):
+                 walltime="00:20:00",
+                 cmd_timeout=10):
```

Comment:
my gut says this should be more like 30 seconds or a minute: the aim is to catch failures where the commands hang for a long long time, and I've definitely seem some queue systems take a very long time.

Reply:
I agree, I actually just had to update my instance's to 60 seconds to not time out

Reply:
in that case, make it even higher than 60 seconds by default -- the default (in my opinion) should be a decent amount (twice?) the time people are experiencing on reasonable systems.